property_panel/components/IntentButtons.js

```diff
@@ -23,17 +23,40 @@ const IntentButtons = ({ selectedElements, onClearIntentsClick, onAddIntentClick
     <div>
       <button
         className="btn btn-default"
+        onClick={() => onAddResponseClick()}>Create Response
+      </button>
+    </div>
+    <div>
+      <button
+        className="btn btn-default"
         onClick={() => onDeleteClick(selectedElements)}>Delete
       </button>
     </div>
+    <div>
+      Conversation Components
+      <button
+        className="btn btn-default"
+        onClick={() => onAddConversationComponent(1)}>Yes-No Questions
```

Comment:
It would be more readable if we use constant variables such as YN_QUESTION = "YES_NO" and WH_QUESTION = "WH" when assigning component types. It's okay now, since we don't have much time.